samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -44,10 +54,15 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
 
   const [chosenCallOption, setChosenCallOption] = useState<IChoiceGroupOption>(callOptions[0]);
   const [teamsLink, setTeamsLink] = useState<TeamsMeetingLinkLocator>();
+  const [alternativeCallerId, setAlternativeCallerId] = useState<string | undefined>();
```

Comment:
nit:
`const [alternativeCallerId, setAlternativeCallerId] = useState<string>();` will work too.

`useState<string>()` means that it can be a string or undefined. :)